packages/puppeteer-core/src/common/Input.ts

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-import {assert} from '../util/assert.js';
-import {CDPSession} from './Connection.js';
-import {_keyDefinitions, KeyDefinition, KeyInput} from './USKeyboardLayout.js';
-import {Protocol} from 'devtools-protocol';
-import {Point} from './JSHandle.js';
+import { assert } from '../util/assert.js';
```

Comment:
I guess custom formatting is done by your editor/ide? could you please revert it so that it's easier to review? `npm run format` should do that.

Reply:
ok, done

Reply:
still shows up wrong for me